test/scenarios.py

```diff
@@ -6983,4 +6968,30 @@
             ),
         },
     ),
+    Scenario(
+        "scenario34",
+        "Read/Write to database multiple times to exercise the concurrency",
+        {
+            "ren": ActionList(
+                [
+                    action.Start("172.20.0.5"),
+                    action.Repeated(
+                        [
+                            action.Parallel(
+                                [
+                                    action.NewTransfer(
+                                        "172.20.0.15", ["/tmp/testfile-small"]
+                                    ),
+                                    action.PurgeTransfersUntil(int(time.time() + 10)),
+                                    action.AssertTransfers([]),
```

Comment:
The `AssertTransfers` should be executed at the very end. In here it can catch the transfer not being deleted from the other parallel run. And it actually showed up in the CI run

Reply:
You're definitely right!